src/SFA.DAS.Commitments.Database/StoredProcedures/GetStatistics.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE PROCEDURE [dbo].[GetStatistics]
+AS
+SELECT
+(
+	SELECT COUNT(Id)
+	FROM [Commitment]
+) AS TotalCohorts,
+(
+	SELECT COUNT(Id)
+	FROM [Apprenticeship]
+) AS TotalAprrenticeships,
+(
+	SELECT COUNT(Id)
+	FROM [Apprenticeship]
+	WHERE PaymentStatus = 1
```

Comment:
I'm not sure on what frequency/schedule the endpoint is likely to be called, but with performance in mind, is it worth adding an index to cover this? - I don't think there is one (but not sure).